scripts/suite.py

```diff
@@ -174,7 +174,7 @@
  --force-priority             Skip the priority check.
 
 """.format(
-    default_machine_type=config.default_machine_type,
+    teuth_machine_type=config.teuth_machine_type,
```

Comment:
Why just not reuse default_machine_type and got to tautology config and set it to smithi?

Reply:
I tried it, but it was getting updated to null looking into the logs; I suppose something to do with the environment as I don't see anything else mangling default_machine_type ¯\_(ツ)_/¯
```
ideepika@teuthology:~/teuthology$ git grep "default_machine_type"
docs/laptop/README.md:default_machine_type: vps
docs/laptop/teuthology.yaml:default_machine_type: vps
docs/siteconfig.rst:    default_machine_type: awesomebox
scripts/suite.py:                              Machine type [default: {default_machine_type}]
scripts/suite.py:    default_machine_type=config.default_machine_type,
teuthology/run.py:                                      teuth_config.default_machine_type)
teuthology/suite/__init__.py:        conf.machine_type = config.default_machine_type
```

Reply:
needs to figure out why it is getting nulled

Reply:
So could you please drop this change and give it a try with the *default_machine_type*?

Reply:
> So could you please drop this change and give it a try with the _default_machine_type_?

yep yep sure, will get back to it.

Reply:
@kyr I think I know the root: https://github.com/ceph/ceph-cm-ansible/pull/614 but I think it could be tested once this PR gets merged

Reply:
> @Kyr I think I know the root: [ceph/ceph-cm-ansible#614](https://github.com/ceph/ceph-cm-ansible/pull/614) but I think it could be tested once this PR gets merged

As far as I know @djgalloway does not manage the /etc/teuthology.yaml via ceph-cm-ansible.

Because many data modified their manually probably.

But I guess I know why it did not work for you in first place, maybe because the system config has this line:
```
default_machine_type: null 
```